anima/env/mayaEnv/publish.py

```diff
@@ -933,6 +969,61 @@ def check_empty_shapes(progress_controller=None):
         )
 
 
+@publisher('model')
+def check_default_uv_set(progress_controller=None):
+    """All mesh have default UVSet map1
+
+    checks if all mesh have 1 default UVSet named as map1
+    """
+    if progress_controller is None:
+        progress_controller = ProgressControllerBase()
+
+    # skip if this is a representation
+    v = staging.get('version')
+    if v and Representation.repr_separator in v.take_name:
+        progress_controller.complete()
+        return
+
+    all_meshes = pm.ls(type='mesh')
+    progress_controller.maximum = len(all_meshes)
+    nodes_with_non_default_uvset = []
+    for node in all_meshes:
+        if len(node.getUVSetNames()) != 1 or node.getUVSetNames()[0] != 'map1':
```

Comment:
no, models can have more than one UV sets. But it is a good idea to check if ``map1`` exists:

```python
if 'map1' in node.getUVSetNames():
```